pkg/pillar/cmd/nim/nim.go

```diff
@@ -36,6 +36,11 @@ const (
 	// Time limits for event loop handlers; shorter for nim than other agents
 	errorTime   = 60 * time.Second
 	warningTime = 40 * time.Second
+
+	// PbrLocalDestPrio : IP rule priority for packets destined to locally owned addresses
+	PbrLocalDestPrio = 12000
+	// PbrLocalOrigPrio : IP rule priority for locally generated packets
+	PbrLocalOrigPrio = 15000
```

Comment:
How about putting all of the Pbr Prio definitions in one file to make it easier to get an overview?

Reply:
Will move all to devicenetwork/pbr.go